sense-android-library/src/nl/sense_os/platform/SensePlatform.java

```diff
@@ -108,108 +88,10 @@ public SensePlatform(Context context) {
     public SensePlatform(Context context, ServiceConnection serviceConnection) {
         mServiceConnection = new SenseServiceConn(serviceConnection);
         mContext = context;
-        trivialSensorRegistrator = new TrivialSensorRegistrator(mContext);
         bindToSenseService();
     }
 
     /**
-     * Convenience method to add a data point without specifying the device UUID
```

Comment:
@tedschmidt i think some module in aim-android-sdk still using this convenience method. Maybe we should annotate this method as deprecated before actually removing it?